js/background.js

```diff
@@ -102,6 +134,20 @@ function Background() {
 
 var background = new Background();
 
+chrome.alarms.create({periodInMinutes: 1});
```

Comment:
The first param to `create` is an optional name. 
https://developer.chrome.com/apps/alarms#method-create

I would give this alarm a name and then in the listener check to see that we're only responding to that alarm.